.gitignore

```diff
@@ -1,4 +1,3 @@
-node_modules
```

Comment:
I thought we were committing the built artifacts? Why do we need the node_modules?

Reply:
For the lambda functions, the `node_modules` directory is the built artifacts. `/dev-portal/node_modules` is not necessary to commit, but `/lambdas/**/node_modules` is.

Reply:
Chatted offline. In the interest of time, we'll leave it as is. As part of upgrading the libraries, I'll change this to `/node_modules` since the root directory isn't strictly needed.